apps/jobs/utils/validators.py

```diff
@@ -119,11 +141,16 @@ def resume_validation(self, resume_file):
                     ) == "D0CF11E0A1B11AE1".encode("ASCII"):
                         upload_file_to_storage = True
                 else:
+                    validationClass.logger.error(
```

Comment:
Log messages and exception messages should be the same. 
You have written "name isn't appropriate", and the method returns a tuple with message "Result file type is not supported"